test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,15 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  test("returns string of names, seperated by commas and an ampersand", () => {
+    expect(
+      joinNames([{ name: "Bart" }, { name: "Lisa" }, { name: "Maggie" }])
+    ).toEqual("Bart, Lisa & Maggie");
+    expect(
+      joinNames([{ name: "Alison" }, { name: "Emily" }, { name: "Jessie" }])
+    ).toEqual("Alison, Emily & Jessie");
+    expect(joinNames([{ name: "Alison" }, { name: "Jessie" }])).toEqual(
+      "Alison & Jessie"
```

Comment:
This is all great stuff. It might be easier to read if you have your arguments saved as variables above the test case when they start getting long. Just for readability though. You could also save the person objects and then just have something like `const bart = { name: "Bart" }` and then use as your argument `[bart, maggie]`